lte/protos/session_manager.proto

```diff
@@ -785,6 +796,23 @@ message SetSMSessionContextAccess {
   RatSpecificContextAccess   rat_specific_context = 2;
 }
 
+
+message UPFSessionState {
+  string imsi = 1;            
+  int32 sess_ver_no = 2;      // Successful Version number snapshot from UPF. If its not the expected parse Failure report.
+}
```

Comment:
isn't the agreement was to send the local tunnel id to identify the session?

Reply:
Ulas. That's right (reference : #2971). Will sync up with SMF team if they can update this.